src/session/content/message_row/label.rs

```diff
@@ -31,7 +30,7 @@ mod imp {
     "#)]
     pub(crate) struct MessageLabel {
         pub(super) text: RefCell<String>,
-        pub(super) indicators: OnceCell<MessageIndicators>,
+        pub(super) indicators: RefCell<Option<MessageIndicators>>,
```

Comment:
The change of making this property an Option can be split in a separate commit (I forgot to do that in the first place).